tsdb/head.go

```diff
@@ -1733,7 +1733,7 @@ func (h *Head) Close() error {
 }
 
 // String returns an human readable representation of the TSDB head. It's important to
-// keep this function in order to avoid the struct dump when the head is stringified in
+// keep this function in order to avoid the struct dump when the head is stringfield in
```

Comment:
stringified is not perfect, but it is the meaning: We could expand that to "converted to string".

stringfield is not correct.

Reply:
Just to confirm: stringified is a correct word so I would not change it: https://en.wiktionary.org/wiki/stringified